cookiecutter/{{cookiecutter.github_repo_name}}/CONTRIBUTING.md

```diff
@@ -204,3 +204,30 @@ These are the steps to create a new release:
    eventually too).
 
 7. Celebrate!
+
+##  Cross-Arch Testing
+
+This project has built-in support for testing across multiple architectures.
+Currently, our CI conducts tests on `arm64` machines using QEMU emulation. We
+also have the flexibility to expand this support to include additional
+architectures in the future.
+
+This project containers Dockerfiles that can be used in the CI to test the
```

Comment:
My comment needed to be applied to all occurrences :angel: 
```suggestion
This project contains Dockerfiles that can be used in the CI to test the
```

Reply:
Right... :facepalm: